test/built-ins/Number/prototype/S15.7.3.1_A3.js

```diff
@@ -5,13 +5,12 @@
 info: Number.prototype value is not +0
 es5id: 15.7.3.1_A3
 description: Checking value of Number.prototype property
-includes: [$FAIL.js]
 ---*/
 
 //CHECK#1
 try {
   (Number.prototype != 0);
-  $FAIL('#1: "(Number.prototype != 0);" lead to throwing exception. Actual: '+(Number.prototype != 0));
```

Comment:
I imagine this is not going to be too friendly of a reported error as $ERROR will throw, causing the check in the catch block to fail as well. But, seems ok for now, it's not like $FAIL was helping anyway.